users/templates/users/user_profile.html

```diff
@@ -20,10 +20,12 @@
 
                     {% if posts %}
                         {% for post in posts|dictsortreversed:'created_at' %}
-                            {% if forloop.counter != 1 %}
-                                <hr/>
+                            {% if post.extra.can_be_seen %}
```

Comment:
That's irrelevant in my opinion. The point is to control who can see the posts and who doesn't, or am I seeing something wrong?